template-for-repository/proofs/Makefile.common

```diff
@@ -233,6 +235,9 @@ HARNESS_GOTO ?= $(GOTODIR)/$(HARNESS_FILE)
 PROJECT_GOTO ?= $(GOTODIR)/$(PROJECT)
 PROOF_GOTO ?= $(GOTODIR)/$(PROOF)
 
+ARPA_BLDDIR ?= $(PROOFDIR)/arpa_cmake
```

Comment:
I would make `arpa_cmake` a variable that can be overridden.  Also, you probably want to use `$(abspath ...)` to ensure that this is an absolute path, you require that in your invocation of cmake below.